andrew/euler/euler_9.py

```diff
@@ -0,0 +1,23 @@
+'''
+There exists exactly one Pythagorean triplet for which a + b + c = 1000.
+Find the product abc.
+'''
+
+
+import math
+
+def pythagoras(a, b):
+    '''Calculate hypotenuse value'''
```

Comment:
Rather than commenting that this function calculates the hypotenuse and calling it pythagoras. Get rid of the comment, and call the function hypotenuse instead.